src/app/main/main.html

```diff
@@ -1,5 +1,4 @@
-<div ng-controller="MainCtrl">
-  <div class="flex-root vflex full-width full-height">
+<div ng-controller="MainCtrl" class="flex-root vflex full-width full-height">
     <div class="full-width no-shrink">
```

Comment:
Once I get a go-ahead to merge on this PR I will add a whitespace commit that resolves the indentation inconsistency—I left it this way to make reviewing easier